python/dockerfile

```diff
@@ -1,4 +1,22 @@
-FROM alpine
+# docker build -t build2022/app:latest -f python/dockerfile .
+# docker run -d -p 9000:9000 build2022/app:latest
+# docker-compose up
+# docker-compose down
+# docker-compose down --rmi all -v # to remove all volumes and images
```

Comment:
Better we removed these lines.

Reply:
Thank you! I will remove all comments now